components/dialogs/AuthDialogContent/Register.vue

```diff
@@ -131,6 +131,7 @@
 </template>
 
 <script>
+import axios from 'axios';
```

Comment:
Does `import axios from '~/plugins/axios'` also work? if yes should prefer that instance

Reply:
that instance has predefined baseURL point to server which is not the use case

Reply:
The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'.